src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -66,7 +66,7 @@ public int Run(FileWorkItemsOptions options, IFileSystem fileSystem)
                 filingContext.DataToInsert = options.DataToInsert.ToFlags();
             }
 
-            var filer = new SarifWorkItemFiler(filingContext);
+            var filer = new SarifWorkItemFiler(filingContext.HostUri, filingContext);
```

Comment:
>HostUri [](start = 61, length = 7)

Why the extra argument if it's part of the context? (Might become clear when I read the rest of the code.)

Reply:
Ok, I see that it's so can create a filer with just a URI, you don't need a context (defaults to null).

---
In reply to: [395181801](https://github.com/microsoft/sarif-sdk/pull/1818#discussion_r395181801) [](ancestors = 395181801)

Reply:
Yes, this is actually a kludgy design decision, not sure it was a good idea. 

---
In reply to: [395218699](https://github.com/microsoft/sarif-sdk/pull/1818#discussion_r395218699) [](ancestors = 395218699,395181801)